Sezai/index.js

```diff
@@ -0,0 +1,103 @@
+//1.1. **Addition.**  Write a program to add up the numbers 1 to 20
+
+for(let i = 0; i<=20; i++){
+    console.log(i);  }
+
+//2. **There are i bottles of beer on the wall. Write a program that will output, "There is one bottle of beer on the wall." "There are two bottles of beer on the wall" up until there are five bottles. 
+
+for(let i=1; i<=5; i++){
+    console.log(`There is ${i} bottle of beer on the wall`)
```

Comment:
You have to you use if statement here! And print up until there are five bottles.